railties/lib/rails/application/bootstrap.rb

```diff
@@ -78,10 +77,6 @@ module Bootstrap
       initializer :bootstrap_hook, group: :all do |app|
         ActiveSupport.run_load_hooks(:before_initialize, app)
       end
-
-      initializer :set_secrets_root, group: :all do
-        Rails::Secrets.root = root
```

Comment:
Since we're passing path when initializing `Rails::Secrets` then it is not needed anymore.
Unless I don't know about some case where it might be needed?